# PositionFeedback.msg
## Positional feedback from sensors. In m/s², rad/s² and seconds

time stamp

float32 imu_ax
float32 imu_ay
float32 imu_az
float32 imu_gx
float32 imu_gy
float32 imu_gz
float32 imu_rx
float32 imu_ry
float32 imu_rz
int64 enc_1
int64 enc_2
int64 enc_3
int64 enc_4
float32 v_enc_1
float32 v_enc_2
float32 v_enc_3
float32 v_enc_4
